### PR TITLE
Adds cocoa interaction for felinids.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
The original idea: https://github.com/AetherStation/AetherStation13/pull/292
Since cats can eat carps without dying, why should they be capable to eat chocolate normally?
It has chances to adjust disgust (mood debuff and puking), message that you're dying and to deal a little bit of organ damage, which i think will be able to kill you only if you have shittonne of it.
## Why It's Good For The Game
New way to bully felinids.
## Changelog
:cl:
add: You can now bully felinids with chocolate (almost not fatal)
/:cl:

### CODE REVIEW

**Comment by Jackraxxus** [comment]
:|

**Comment by JohnFulpWillard** [comment]
Relevant PR: https://github.com/tgstation/tgstation/pull/48066

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Requires oranges approval

**Comment by Farquaar** [comment]
Reject modernity

**Comment by Helg2** [comment]
@optimumtact Reject modernity. No one sits on Reddit.

File: code/modules/mob/living/carbon/human/species_types/felinid.dm
Comment (tralezab) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:25]: species inherent? this sucks. why not stomach inherent instead? genetic trait applied to felinids that modifies their stomach to do it? but i hate species dependent interactions over organ and mutation ones

File: code/modules/mob/living/carbon/human/species_types/felinid.dm
Comment (tralezab) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:30]: well, this i hate. anyone adds a chocolate drink, reagent, anything and they have to remember to update this increasingly large list of chocolate types

**Comment by Fikou** [comment]
Species changes require prior maintainer approval

**Review by tralezab — COMMENTED** [review]
honestly? i'd be fine with it conceptually. codewise, i just can't get behind it

File: code/modules/mob/living/carbon/human/species_types/felinid.dm
Comment (CRITAWAKETS) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm:30]: the ideal way to fix it is to add a new food category: chocotastic

Comment (Helg2) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm:30]: So they puke because of chocolate cake which doesn't have any cocoa?

**Comment by itseasytosee** [comment]
@Helg2 I can show you how to make this stomach specific hmu

**Comment by optimumtact** [comment]
no point, i'm not  interested in this change

**Pull request closed without merging**
